client/components/marketing-survey/cancel-purchase-form/step-components/upsell-step.tsx

```diff
@@ -157,8 +157,8 @@ export default function UpsellStep( { upsell, site, purchase, ...props }: StepPr
 					acceptButtonText={ translate( 'Get help building my site' ) }
 					onAccept={ () => {
 						recordTracksEvent( 'calypso_cancellation_upsell_step_buily_by_click' );
+						window.location.replace( builtByURL );
```

Comment:
Noting that if we want to leave a mark in the user's history, we might want to use `window.location.href` instead. `.replace` replaces the current history entry with the new URL.

Reply:
afaik `replace` doesn't force a fresh server request

Reply:
Seems to work on production